gems/smithy/spec/fixtures/endpoints/endpoint-bindings.smithy

```diff
@@ -0,0 +1,346 @@
+$version: "2.0"
```

Comment:
I think this file is added in by mistake? You already have model.smithy under endpoint-bindings